hooks/useQuoter.ts

```diff
@@ -5,7 +5,7 @@ import { Oracle, Quoter } from '../types/contracts'
 import oracleAbi from '../abis/oracle.json'
 import { QUOTER_CONTRACT } from '../constants/contracts'
 
-const useQuoter = () => {
+export const useQuoter = () => {
```

Comment:
Why don't we just use useContractReads for the quoter?

Reply:
Depends on where we use quoter. If it's inside a react component or another hook we can user useContractReads. 
But if we are using it outside react (plain ts util files) you can't use hooks there!

Reply:
ah I see!